### PR TITLE
[Outlook] (contextless) Create article

### PR DESCRIPTION
**Pull request by samantharamon**
Related PR: https://github.com/OfficeDev/office-js-docs-reference/pull/1573

### CODE REVIEW

File: docs/outlook/contextless.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+title: Activate your Outlook add-in without the Reading Pane enabled or a message selected (preview)
+description: Learn how to activate your Outlook add-in without enabling the Reading Pane or first selecting a message.
+ms.date: 04/21/2023
+ms.topic: how-to
+ms.localizationpriority: medium
+---
+
+# Activate your Outlook add-in without the Reading Pane enabled or a message selected (preview)
+
+With a simple manifest configuration, Outlook add-ins created for the Message Read surface can now activate a task pane without the Reading Pane enabled or a message first selected from the mailbox. Follow the walkthrough to learn more and unlock additional capabilities for your add-in, such as enabling users to access content from different data sources, such as OneDrive or a customer relationship management (CRM) system, directly from their Outlook client.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/contextless.md:11]: you use "such as" twice in the same sentence - consider rewording

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Looks great

**Review by lindalu-MSFT — APPROVED** [review]
Minor suggestions! Looks good.

File: docs/outlook/contextless.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+---
+
+# Activate your Outlook add-in without the Reading Pane enabled or a message selected (preview)
+
+With a simple manifest configuration, Outlook add-ins created for the Message Read surface can now activate a task pane without the Reading Pane enabled or a message first selected from the mailbox. Follow the walkthrough to learn more and unlock additional capabilities for your add-in. For example, you can enable your users to access content from different data sources, such as OneDrive or a customer relationship management (CRM) system, directly from their Outlook client.
+
+> [!IMPORTANT]
+> Features in preview shouldn't be used in production add-ins. We invite you to test this feature in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
+
+> [!NOTE]
```
[the remark this answers is not in the document]
Comment (samantharamon) [reply docs/outlook/contextless.md]: Thanks for the feedback, @lindalu-MSFT. I moved the second alert to the "Configure the manifest" section.

**Pull request merged**